README.md

```diff
@@ -1 +1,95 @@
-# cars-bnb-frontend
\ No newline at end of file
+# Cars BnB App
+
+## Welcome! 👋
```

Comment:
- [ ] FRONTEND:  I could get to the first screen and it's very beautiful.  Let's show off your front-end app.  Kindly add a screenshot to your readme so everybody can view it.  💪